tests/test_core.py

```diff
@@ -1052,14 +1090,15 @@ def test_setjmp_many_2(self):
 
     self.do_run(src, r'''d is at 24''')
 
+  @with_both_sjlj_handling
   def test_setjmp_noleak(self):
     self.do_runf(test_file('core/test_setjmp_noleak.c'), 'ok.')
 
   @with_both_exception_handling
   def test_exceptions(self):
     self.set_setting('EXCEPTION_DEBUG')
     self.maybe_closure()
-    for support_longjmp in [0, 1]:
```

Comment:
Revert all the changes here and below in this file?

Reply:
Done